prov/verbs/src/verbs_mr.c

```diff
@@ -73,8 +73,8 @@ static struct ibv_mr *vrb_reg_hmem_dmabuf(enum fi_hmem_iface iface,
 		[FI_HMEM_CUDA] = TRY,
 		[FI_HMEM_ROCR] = TRY,
 		[FI_HMEM_ZE] = TRY,
+		[FI_HMEM_SYNAPSEAI] = TRY,
```

Comment:
Does synapseai have peer_mem driver to support the retry?

Reply:
You are right, we don't want to fallback to this retry mechanism. I fixed it.